src/app/app.module.ts

```diff
@@ -62,41 +63,45 @@ const appRoutes: Routes = [
     VoteToPercentPipe,
     TopBilledCastCardComponent
   ],
-	imports: [
-		BrowserModule,
-		HttpClientModule,
-		FormsModule,
-		AngularSvgIconModule.forRoot(),
-		RouterModule.forRoot(appRoutes),
-		BrowserAnimationsModule,
-		MatSlideToggleModule,
-		MatCardModule,
-		MatGridListModule,
-		// NgCircleProgressModule,
-		NgCircleProgressModule.forRoot({
-			// set defaults here
-			radius: 100,
-			outerStrokeWidth: 16,
-			innerStrokeWidth: 8,
-			outerStrokeColor: "#78C000",
-			innerStrokeColor: "#C7E596",
-			animationDuration: 300
-		}),
-		YouTubePlayerModule,
-		MatInputModule,
+  imports: [
+    BrowserModule,
+    HttpClientModule,
+    FormsModule,
+    AngularSvgIconModule.forRoot(),
+    RouterModule.forRoot(appRoutes),
+    BrowserAnimationsModule,
+    MatSlideToggleModule,
+    MatCardModule,
+    MatGridListModule,
+    // NgCircleProgressModule,
```

Comment:
Try not to leave commented code in branches. Unless it's comment thet gives some info like on 78 line: 
`// set defaults here`

Reply:
ok, done